importer/importer.py

```diff
@@ -166,6 +176,11 @@ def get_items(connection,
         class_to_use = SPECIFIC_TABLES[specific_table_name]["class"]
         data_files = load_data_files(
             SPECIFIC_TABLES[specific_table_name]["data_files"])
+        if SPECIFIC_TABLES[specific_table_name]["subclass_downloads"]:
+            for dataset_title in SPECIFIC_TABLES[specific_table_name]["subclass_downloads"]:
```

Comment:
use `for subclass_title, subclass_item in specific_table.get["subclass_downloads"].items()` that way you have both the key and the value available for later calculations.